.coveragerc

```diff
@@ -1,7 +1,25 @@
 [run]
+source_pkgs =
+	setuptools
+	pkg_resources
 omit =
 	# leading `*/` for pytest-dev/pytest-cov#456
-	*/.tox/*
+	*/_vendor/*
+	*/_distutils/*
+	*/tests/*
```

Comment:
I normally prefer to keep coverage reports on the tests, as I'd expect tests to be covered as well, and uncovered code in tests is usually a sign of a problem as well.

Reply:
exactly: https://nedbatchelder.com/blog/201908/dont_omit_tests_from_coverage.html

Reply:
Thank you for the feedback, I added the tests back in my latest commits.